src/Components/Intro/Intro.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './Intro.css';
+import {faChartLine} from "@fortawesome/free-solid-svg-icons";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import {useHistory} from 'react-router-dom';
+
+function Intro(props) {
+    const history=useHistory()
+    const onStartClick = () => {
+      history.push("/linear");
+    }
+    return (
+        <div style={{backgroundColor: '#FFFFFF'}}>
+
+          <header>
+            <h1 className="introHeader">Linear vs. Logistic Regression</h1>
+          </header>
+
+          <div>
+            <p id="Digression">
+              Digression on Regression
+            </p>
+
+            <div>
+              <Grid container spacing={2}>
+                <Grid item xs={7}>
+                  <p id="intro">
+                    What is linear and logistic regression? 
+                    Why does it matter? This lesson will give you an introduction to 
+                    linear and logistic regression and explain why it is such an important topic. 
+                  </p> 
+                </Grid>
+                <Grid item xs={3}>
+                  <FontAwesomeIcon id="chart-icon" icon={faChartLine}/>
+                </Grid>
+              </Grid>
+            </div>
+
+            <div style={{textAlign: 'center', marginTop: "0", marginBottom: "2em"}}>
+              <Button onClick={onStartClick} variant="contained"
+                style={{
+                  backgroundColor: "#95CF67",
+                  color: "white",
+                  fontSize: "2em"
+                }}>
+                Start
+              </Button>
```

Comment:
We should generally avoid using inline styling where we can. This will have to be put into another PR, but these buttons will be changed to use Material UI's [palette](https://material-ui.com/customization/palette/). Same goes for the `.Green-button` class in [LinearGoal.css](https://github.com/uclaacm/digression-on-regression/pull/1/files/0fcec04350e4acd67d58acac7c97c85f3c60dfd2#diff-8fd6e2d8f6c6d1b97d267e50f64f8ea8R13).